cmd/contour/servecontext.go

```diff
@@ -172,6 +174,38 @@ func newServeContext() *serveContext {
 // TLSConfig holds configuration file TLS configuration details.
 type TLSConfig struct {
 	MinimumProtocolVersion string `yaml:"minimum-protocol-version"`
+
+	// FallbackCertificate defines the namespace/name of the Kubernetes secret to
+	// use as fallback when a non-SNI request is received.
+	FallbackCertificate `yaml:"fallback-certificate,omitempty"`
```

Comment:
```suggestion
	FallbackCertificate FallbackCertificate  `yaml:"fallback-certificate,omitempty"`
```

We should name the field here, otherwise `Name` and `Namespace` are accessible as fields on a `TLSConfig` variable, which is confusing.